apps/aehttp/src/aehttp_api_validate.erl

```diff
@@ -192,6 +201,7 @@ prepare_param_({"type", "float"}, Value, Name, _, _) ->
 prepare_param_({"type", "integer"}, Value, Name, _, _) ->
     try {ok, to_int(Value)}
     catch
+        error:function_clause  -> 1 = {domat, Name, Value};
```

Comment:
wut? This will never match

Reply:
oops, one of my debug messages leaked out :D